client/components/resource-details/ResourceInfo.tsx

```diff
@@ -144,29 +137,164 @@ const Reviews = ({ reviews }: ReviewsProps) => {
   return (
     <div>
       <h1>Reviews</h1>
-      {reviews.map((review) => {
-        return <ReviewEntry review={review} />
+      {reviews.map((review, index) => {
+        return <ReviewCard key={index} review={review} />
       })}
     </div>
   )
 }
 
 const Avatar = styled.img`
-  border-radius: 4px;
+  border-radius: 50%;
   display: block;
-  margin-top: 40px;
-  height: 100px;
-  width: 100px;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-top: 24px;
+  height: 80px;
+  width: 80px;
+`
+
+const RatingList = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  flex-direction: column;
+  margin: 24px;
+  margin-left: auto;
+  align-self: flex-start;
+  @media ${smallerThan(width.mobileS)} {
+    margin-left: 24px;
+  }
+`
+
+const ReviewTopRow = styled.div`
+  display: flex;
+  @media ${smallerThan(width.laptop)} {
+    flex-wrap: wrap;
+  }
+`
+
+const ShowOnLaptop = styled.div`
+  @media ${smallerThan(width.laptop)} {
+    display: none;
+  }
 `
 
-const Rating = styled.p``
+const HideOnLaptop = styled.div`
+  @media ${largerThan(width.laptop)} {
+    display: none;
+  }
+`
+
+const AuthorInfo = styled.div`
+  font-style: italic;
+`
 
-const ReviewEntry = ({ review }: { review: Review }) => {
+const Comment = styled(({ className, review }: { className?: string; review: Review }) => {
   return (
-    <>
-      <Avatar />
-      <Description />
-      <Rating />
-    </>
+    <div className={className}>
+      <Description>{review.comment}</Description>
+      <AuthorInfo>
+        — {Faculty[review.faculty]} Student, {timeSince(review.timestamp)} ago
+      </AuthorInfo>
+    </div>
   )
+})`
+  margin: 24px;
+`
+
+//https://stackoverflow.com/questions/3177836/how-to-format-time-since-xxx-e-g-4-minutes-ago-similar-to-stack-exchange-site
+function timeSince(date: Date): string {
+  var seconds = Math.floor((Date.now() - Number(date)) / 1000)
+
+  var interval = seconds / 31536000
+
+  if (interval > 2) {
+    return Math.floor(interval) + ' years'
+  }
+  interval = seconds / 2592000
+  if (interval > 2) {
+    return Math.floor(interval) + ' months'
+  }
+  interval = seconds / 86400
+  if (interval > 2) {
+    return Math.floor(interval) + ' days'
+  }
+  interval = seconds / 3600
+  if (interval > 2) {
+    return Math.floor(interval) + ' hours'
+  }
+  interval = seconds / 60
+  if (interval > 2) {
+    return Math.floor(interval) + ' minutes'
+  }
+  return Math.floor(seconds) + 2 + ' seconds'
```

Comment:
There's a library for that
https://date-fns.org/v2.22.1/docs/formatRelative

Reply:
Replaced this function with date-fns formatRelative, resolved